server/views/azuread-b2c-login.html

```diff
@@ -144,6 +164,13 @@
       <div id="api"></div>
     </div>
 
+    <div class="govuk-width-container">
+      <div class="outro">
+        This sign in page is for users who do not work for HMCTS.<br />
+        <a id="dedicatedPage" href="#">There is a dedicated page for HMCTS employees to sign into the portal.</a>
```

Comment:
Should this link somewhere?

Reply:
I thought maybe we didn't know the link yet, could you please create a ticket so we don't forget to set it up.

Reply:
https://tools.hmcts.net/jira/browse/DMP-621